src/service/resource.js

```diff
@@ -626,6 +626,9 @@ export class Resource {
    * Returns a previously measured layout box adjusted to the viewport. This
    * mainly affects fixed-position elements that are adjusted to be always
    * relative to the document position in the viewport.
+   * The returned layoutBox is
```

Comment:
Styling nits:

- Add a `:` to the end of this line
- Prefix each point with a `- `

Reply:
done